accounts/src/main/java/eu/kevin/accounts/bankselection/usecases/ValidateBanksConfigUseCase.kt

```diff
@@ -0,0 +1,54 @@
+package eu.kevin.accounts.bankselection.usecases
+
+import eu.kevin.accounts.networking.KevinAccountsClient
+import eu.kevin.accounts.networking.entities.ApiBank
+
+class ValidateBanksConfigUseCase(
+    private val accountsClient: KevinAccountsClient
+) {
+
+    suspend fun validateBanksConfig(
+        authState: String,
+        country: String?,
+        preselectedBank: String?,
+        banksFilter: List<String>,
+        requireAccountLinkingSupport: Boolean
+    ): Status {
+        val apiBanks = accountsClient.getSupportedBanks(authState, country).data
```

Comment:
Right now we would be making this api call always when Account/Payment is initiated even if preselected bank or filters were not provided at all.

I think it should only check when either of them is present?